components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityTenantUtil.java

```diff
@@ -422,6 +422,14 @@ public static boolean isTenantedSessionsEnabled() {
         return Boolean.parseBoolean(IdentityUtil.getProperty(IdentityCoreConstants.ENABLE_TENANTED_SESSIONS));
     }
 
+    /**
+     * Checks if it is required to specify carbon.super in tenant qualified URLs.
+     * @return true if it is mandatory, false otherwise
+     */
```

Comment:
```suggestion
    /**
     * Checks if it is required to specify carbon.super in tenant qualified URLs.
     * 
     * @return true if it is mandatory, false otherwise
     */
```

Reply:
Addressed with commit d503a1de9b2d2b22915028e34573bbb5ca25663f